website/source/api/auth/jwt/index.html.md

```diff
@@ -102,8 +102,8 @@ entities attempting to login. At least one of the bound values must be set.
 ### Parameters
 - `name` `(string: <required>)` - Name of the role.
 - `role_type` `(string: <optional>)` - Type of role, either "oidc" (default) or "jwt".
-- `bound_audiences` `(array: <required>)` - List of `aud` claims to match
```

Comment:
Suggest instead: 

`bound_audiences` `(array: <optional>)` - Required for "jwt" roles, optional for "oidc" roles.  List of `aud` claims to match against. Any match is sufficient. Any match is sufficient. 

That way jwt people skimming / reading too quickly are less likely to overlook it.

But your way is fine too.

Reply:
I authored it that way at first, but decided the current way is OK since: a) we expect more OIDC use now that it's available  b) if they omit it during the JWT role creation, they'll get an error saying it's required.